### PR TITLE
fix(zh-cn/whitespace): fix a content problem

### PR DESCRIPTION
**Pull request by zb81**
### Description

Fixed a content problem of HTML interpretation.

### Motivation

HTML tags and interpretation are inconsistent.

### CODE REVIEW

File: files/zh-cn/web/api/document_object_model/whitespace/index.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ slug: Web/API/Document_Object_Model/Whitespace
 `<h1>` 元素只包含行内元素，实际上包含：
 
 - 一个文本节点（包含一些空格，单词“Hello”和一些制表符）。
-- 一个行内元素（`<span>`，包含一个空格和单词“Hello”）。
+- 一个行内元素（`<span>`，包含一个空格，一个单词“World”和一个感叹号“!”）。
```
Comment (yin1999) [inline files/zh-cn/web/api/document_object_model/whitespace/index.md:90]: could you also update the content in [en-US](https://github.com/mdn/content/blob/main/files/en-us/web/api/document_object_model/whitespace/index.md?plain=1#L91)

Comment (zb81) [reply files/zh-cn/web/api/document_object_model/whitespace/index.md:90]: Sure, but I found that the content in [en-US]([en-US](https://github.com/mdn/content/blob/main/files/en-us/web/api/document_object_model/whitespace/index.md?plain=1#L91)) is right.

Comment (yin1999) [reply files/zh-cn/web/api/document_object_model/whitespace/index.md:90]: We just translate the en-US content. So this sentence should be "包含一个空格和一个单词“World!”"

Comment (zb81) [reply files/zh-cn/web/api/document_object_model/whitespace/index.md]: Updated.

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**